Explorer/Assets/DCL/StylizedSkybox/Scripts/StylizedSkyboxSettingsAsset.cs

```diff
@@ -0,0 +1,63 @@
+using DCL.AssetsProvision;
+using System;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+
+namespace DCL.StylizedSkybox.Scripts
+{
+    [CreateAssetMenu(menuName = "Create Stylized Skybox Settings", fileName = "StylizedSkyboxSettings", order = 0)]
```

Comment:
Recently we started moving SOs to the DCL folder to avoid having an endless scroll, could you please move it there? :)

Reply:
Should it be DCL/ or SO/ or DCL/SO/? 🤔 
<img width="639" alt="image" src="https://github.com/user-attachments/assets/6b8d2d8b-cf10-46c2-ae8e-f6d549248ca0" />

<img width="637" alt="image" src="https://github.com/user-attachments/assets/d4560717-c7c6-4250-9fb7-ec9c01837ebc" />

Reply:
DCL/SO